keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/sync/LockTests.kt

```diff
@@ -0,0 +1,60 @@
+package com.netflix.spinnaker.keel.sync
+
+import com.netflix.spinnaker.time.MutableClock
+import dev.minutest.junit.JUnit5Minutests
+import dev.minutest.rootContext
+import strikt.api.expectThat
+import strikt.assertions.isFalse
+import strikt.assertions.isTrue
+import java.time.Clock
+import java.time.Duration
+
+abstract class LockTests<T : Lock> : JUnit5Minutests {
```

Comment:
Trying to get a Redis impl to pass these tests which rely on being able to fake the clock.